README.md

```diff
@@ -67,13 +73,25 @@ In order to run this project you need:
 [Github](https://github.com/JoyTaribagshaw/My-Portfolio)
 - Navigate to the clone reppository by running cd `My-Portfolio`
 
+### Install
+
+Install this project with:
+
+```sh
+  cd portfolio-version
+  npm install
+
 
 ### Usage
 
 To run the project, execute the following command:
 
     open the index file in the browser
 
+    ### Run tests
+
+To run tests, click on inspect to open the browser dev tools where you can inspect the entire HTML and CSS code. 
+
 ### Deployment
 
 You can deploy this project using:
```

Comment:
Kindly make sure you are using the correct Micoroverse `README` template. This is somehow different from it.

Reply:
Hello Bantealem, please this is still unclear to me, how do I connect with you on a call? @Bantealem

Reply:
Hello @JoyTaribagshaw, thank you for reaching out to me.
Kindly, use the correct [README template](https://github.com/microverseinc/readme-template) and customize it as you did for "Hello Microverse project"